synapseclient/client.py

```diff
@@ -2106,22 +2022,27 @@ def get_annotations(
         """
         return from_synapse_annotations(self._getRawAnnotations(entity, version))
 
-    @deprecated.sphinx.deprecated(
+    @deprecated(
         version="2.1.0",
-        reason="deprecated and replaced with :py:meth:`set_annotations` "
+        reason="deprecated and replaced with `set_annotations` "
```

Comment:
Do we want to link this? e.g. `[set_annotations][]`

Reply:
Not here. The decorator isn't added to the mkdocs, only the docstring is. Mkdocs had a plug in for autorefs which is responsible for linking to the correct docs page.

This message is printed in the console when running this method.

Reply:
Good to know, thanks